scripts/create-kubestellar-demo-env.sh

```diff
@@ -139,9 +104,83 @@ checking_cluster() {
 
     done
 }
+##########################################
+
+echo -e "\nStarting environment clean up..."
+echo -e "Starting cluster clean up..."
+
+cluster_clean_up "kind delete cluster --name kubeflex"
+cluster_clean_up "kind delete cluster --name cluster1"
+cluster_clean_up "kind delete cluster --name cluster2"
+echo -e "Cluster space clean up has been completed"
+
+echo -e "\nStarting context clean up..."
+
+context_clean_up
+echo "Context space clean up completed"
+
+echo -e "\nStarting the process to install KubeStellar core: kind-kubeflex..."
+
+clusters=(cluster1 cluster2)
+
+for cluster in "${clusters[@]}"; do
+   (
+     echo -e "Creating cluster ${cluster}..."
+     kind create cluster --name "${cluster}" >/dev/null 2>&1 &&
+     kubectl config rename-context "kind-${cluster}" "${cluster}" >/dev/null 2>&1
+     echo -e "${cluster} creation and context setup complete"
+   ) &
+done
+
+wait 
+
+echo -e "Creating KubeFlex cluster with SSL Passthrough"
+curl -s https://raw.githubusercontent.com/kubestellar/kubestellar/v${KUBESTELLAR_VERSION}/scripts/create-kind-cluster-with-SSL-passthrough.sh | bash -s -- --name kubeflex --port 9443 
+# >/dev/null 2>&1
+
+echo -e "Completed KubeFlex cluster with SSL Passthrough"
+
+(docker pull ghcr.io/loft-sh/vcluster:0.16.4 && kind load docker-image ghcr.io/loft-sh/vcluster:0.16.4 --name kubeflex) & 
```

Comment:
Here is a more direct way to get that done:

```console
mspreitz@mjs13 kubestellar % docker exec kubeflex-control-plane crictl pull ghcr.io/loft-sh/vcluster:0.16.4
Image is up to date for sha256:00428133d55e8b3f1b699f390bc4f0dd79e3a2635aad1d0604f6c8df09803166
```

Reply:
Parallelization demo:

```console
mspreitz@mjs13 test4 % docker exec kubeflex-control-plane bash -c "echo a & echo b"
b
a
```

Reply:
this way is much slower than what I have in the script now. keeping the way I have it

Reply:
Ah, the existing way uses a cache on the host machine.